lib/active_record/annotate/file.rb

```diff
@@ -26,13 +27,20 @@ def annotate_with(annotation, configurator)
           @lines.shift
         end
         
-        if configurator.yard?
-          backticks = '# ```'
-          annotation.unshift(backticks).push(backticks)
+        if annotation.first.start_with?("# create_table") || debug ### Ignore Errors
+          if configurator.yard?
+            backticks = '# ```'
+            annotation.unshift(backticks).push(backticks)
+          end
+          
+          @lines.unshift(*annotation, nil)
+        elsif debug
+          ### Print errors if annotate_debug=true is passed
+          puts annotation
```

Comment:
It seems like these lines of code can never be reached - you have `debug` both in `if` and in `elsif`.